libs/ui-lib/package.json

```diff
@@ -101,6 +101,11 @@
     "uuid": "^8.1",
     "yup": "^0.28.3"
   },
+  "overrides": {
+    "@patternfly/react-core": {
+      "attr-accept": "2.2.2"
+    }
+  },
```

Comment:
I don't think we need them here we are no longer installing peer dependencies.
Deps in this section get installed by CIM, assisted-ui or uhc-portal